### ISSUE TITLE
[Bug Report] Hover over players not working in 1.18

### ISSUE THREAD

**Issue by TomLewis** [jpenilla/MiniMOTD#91]
**Describe the bug**
There is no hover over players being shown in 1.18.

**To Reproduce**
1. Use 1.18, and hover over online players number, it shows nothing.

**Expected behavior**
To show random online players

**Server Software and Version:**
 - Vendor: waterfall 1.18 477
 - Minecraft Version: 1.18.1

**Comment by jpenilla** [comment]
MiniMOTD does not modify the player list hover in any way unless the option to disable the hover is enabled. My guess is that either this is a bungee issue, or you have the option to disable the player list hover enabled.

**Comment by TomLewis** [comment]
What would it be in bungee/waterfall to disable it?

**Comment by jpenilla** [comment]
I am referring to `hide-player-count` and `disable-player-list-hover` in the MiniMOTD config

**Comment by TomLewis** [comment]
I have `hide-player-count=false`
`disable-player-list-hover=false`
Any other ideas? Ip is `mc.piratemc.com` if you wanted to have a nose!

**Comment by LudovicAns** [comment]
Yup this is clearly  a bug that need to be fixed. Same config and same bug here.
